docs/book/features/file_volume.md

```diff
@@ -13,7 +13,7 @@ In order to utilize this feature in your vSphere environment, you need to make s
 
 - Enable and configure the file service in your vSAN cluster configuration. You must configure the necessary file service domains, IP pools, network etc in order to create file share volumes. Refer to [vSphere 7.0 vSAN File service](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-82565B82-C911-42F7-85B1-E9EF973EE90C.html) documentation to get started.
 
-- Establish a dedicated file share network connecting all the kubernetes nodes and make sure this network is routable to the vSAN File Share network.  Refer to [Network Access of vSAN File Share](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-EFC00FFF-E720-44F1-B229-4C13687E6B85.html) to understand the setup better.
+- Establish a dedicated file share network connecting all the kubernetes nodes and make sure this network is routable to the vSAN File Share network.  Refer to [Network Access of vSAN File Share](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-EFC00FFF-E720-44F1-B229-4C13687E6B85.html) to understand the setup better. Configure the kubernetes nodes with the same DNS server as the one used to configure the file services in the vSAN cluster configuration.
```

Comment:
DNS configuration on the node can be a separate bullet point. You may want to add some text to explain why the k8s nodes needs to be configured with the same DNS server used to create vSAN file services.

Reply:
Then I will not include this fix in this PR. I assumed it is just a one line addition, so added it here. Will create a separate PR for it.